src/native/fs/fs_napi.cpp

```diff
@@ -145,6 +145,7 @@ set_statfs_res(Napi::Object res, Napi::Env env, struct statfs &statfs_res)
 struct FSWorker : public Napi::AsyncWorker
 {
     Napi::Promise::Deferred _deferred;
+    Napi::ObjectReference _args_ref;
```

Comment:
We can add this to explain:
```suggestion

    // _args_ref is used to keep refs to all the args for the worker lifetime,
    // which is needed for workers that receive buffers,
    // because in their ctor they copy the pointers to the buffer's memory,
    // and if the JS caller scope does not keep a ref to the buffers until after the call,
    // then the worker may access invalid memory...
    Napi::ObjectReference _args_ref;
```